misc/scripts/refcnt_test.sh

```diff
@@ -80,20 +81,21 @@ function check_recovery_record {
 
 function test_crash_recovery {
     timeout=$1
-    echo "Checking recovery for VMDK plugin kill -9"
-    kill -9 `pidof docker-volume-vsphere`
-    until pids=$(pidof docker-volume-vsphere)
+    echo "Checking recovery through docker kill"
+    # kill docker daemon forcefully
+    pkill -9 dockerd
+    until pids=$(pidof dockerd)
     do
-        echo "Waiting for docker-volume-vsphere to restart"
+        echo "Waiting for docker to restart"
         sleep 1
     done
 
     echo "Waiting for plugin init"
-    sleep 3
+    sleep 5
     sync  # give log the time to flush
     wait_for check_recovery_record $timeout
     if [ "$?" -ne 0 ] ; then
-        echo PLUGIN RESTART TEST FAILED. Did not find proper recovery record
+        echo DOCKER RESTART TEST FAILED. Did not find proper recovery record
```

Comment:
This test will in turn test plugin restart (docker restart). Plugin restart through docker restart tests the docker api consumption, timeout and handling of parallel mounts/unmounts till refcounting is successful. Not sure why not have this as automated test?